core/src/main/java/de/danoeh/antennapod/core/util/playback/Playable.java

```diff
@@ -215,11 +215,11 @@ private static Playable createExternalMediaInstance(SharedPreferences pref) {
             return result;
         }
 
-        private static Playable createRemoteMediaInstance(SharedPreferences pref) {
-            //TODO there's probably no point in restoring RemoteMedia from preferences, because we
-            //only care about it while it's playing on the cast device.
-            return null;
-        }
+//        private static Playable createRemoteMediaInstance(SharedPreferences pref) {
+//            //TODO there's probably no point in restoring RemoteMedia from preferences, because we
+//            //only care about it while it's playing on the cast device.
+//            return null;
+//        }
```

Comment:
True, although one has to remember it is there (git history). In this case, this piece of code is totally irrevelant, so let's delete it